pkg/bpf/bpf.go

```diff
@@ -113,6 +113,13 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+const (
+	BPF_MAP_TYPE_HASH     = C.BPF_MAP_TYPE_HASH
```

Comment:
I guess these are exceptions to the CamelCase rule as they mirror constants from linux/bpf.h? Or should I adjust them to CamelCase to make the linter happy?